sentry-kotlin-multiplatform/src/androidMain/AndroidManifest.xml

```diff
@@ -8,7 +8,14 @@
             android:name="io.sentry.android.core.SentryInitProvider"
             android:authorities="${applicationId}.SentryInitProvider"
             android:exported="false"
-            tools:node="remove"/>
+            tools:node="remove">
+        </provider>
+        <provider
+            android:name="io.sentry.kotlin.multiplatform.ContextProvider"
+            android:authorities="${applicationId}.ContextProvider"
```

Comment:
Maybe makes sense to change this to `SentryContextProvider`, to reduce the risk of clashing with any user-defined app authorities.

```suggestion
            android:authorities="${applicationId}.SentryContextProvider"
```